app/controllers/organizations_controller.rb

```diff
@@ -58,13 +63,17 @@ def destroy
   end
 
   def create
-    if current_user.organization_id.present?
-      redirect_to organization_path(current_user.organization_id), alert: "You can only have one organization."
-    end
-    @organization       = Organization.new(organization_params)
-    @organization.user  = current_user
+    @organization      = Organization.new(organization_params)
+    @organization.user = current_user
+
     if @organization.save
-      redirect_to organization_path(@organization), notice: "Organization Created!"
+      FetchOrganizationNewsJob.perform_now(@organization.name, @organization.id)
```

Comment:
Why this has `perform_now` instead of `perform_later`? It should happen in the background